### PR TITLE
Style: Use new color system

### PR DESCRIPTION
**Pull request by VitroidFPV**
In preparation for a larger UI overhaul. Uses variables for all colors that get switched in dark mode, also further consistent with the website. `dark-theme.less` is used for resources that can't be easily/cleanly changed with variables - logos, graphics, and one-off colors. 

It was a rough pass changing everything over, it's possible I missed something even through testing. Please check and report anything odd!

Screenshots below, but it's probably better to look through in the build preview we'll get shortly. Maybe a good place to try the collaborative review feature 😅
image
image
image
image
image

### CODE REVIEW

**Comment by McGiverGim** [comment]
The dark theme is ok. The light theme I'm not sure. Specially the header and the log in the same colour than the rest. I think I prefer some little more of contrast.

**Comment by nerdCopter** [comment]
I find the yellow/orange links on light background difficult to eye.
Also something pretty about the old light theme, although not 100% light.

left this PR PWA  -- right master NWjs
image

File: src/components/port-picker/PortsInput.vue [hunk 1]
```diff
@@ -91,7 +91,7 @@ export default {
     },
 };
 </script>
-<style scoped>
+<style scoped lang="less">
```
Comment (chmelevskij) [inline src/components/port-picker/PortsInput.vue:94]: Do we need `lang="less"`here? Don't think we are using any features from less in vue parts

File: src/css/theme.css
Comment (chmelevskij) [inline src/css/theme.css:1]: Something seems to be odd with the indentation here

**Comment by chmelevskij** [comment]
Isn't the light theme the same as on https://betaflight.com/docs/wiki/release/Betaflight-4-5-Release-Notes? If it is then we should probably just keep it the same, at least for now

**Comment by chmelevskij** [comment]
Though, tried lighthouse on both documentation site and deploy preview. It tends to flag up similar issue with the link colours.

Given that point of this PR is to align docs and betaflight configurator I guess we can live with that. Otherwise we'd be again having different looks between configurator and docs... Would make sense addressing that in separate PR if you ask me.

**Comment by VitroidFPV** [comment]
Closing as this was included in #3956

**Pull request closed without merging**